frontend/src/pages/Interventions/data/actions.ts

```diff
@@ -44,6 +44,7 @@ interface UpdateInterventionAction {
   wordIdx?: number;
   activityIdx?: number;
   interventions?: Interventions;
+  durationInSeconds?: number;
```

Comment:
Is there ever a time where this will be undefined? I wonder if we want this to be optional or not

Reply:
Yes (but keep answer2Correct, and answer3Correct optional since those fields are actually optional)